grails-app/services/de/iteratec/osm/measurement/environment/wptserver/ResultPersisterService.groovy

```diff
@@ -74,13 +74,14 @@ class ResultPersisterService implements iResultListener {
     @Override
     public void listenToResult(
             WptResultXml resultXml,
-            WebPageTestServer wptserverOfResult) {
+            WebPageTestServer wptserverOfResult,
+            long jobId) {
 
         try {
-            checkJobAndLocation(resultXml, wptserverOfResult)
-            persistJobResult(resultXml)
-            persistResultsForAllTeststeps(resultXml)
-            informDependents(resultXml)
+            checkJobAndLocation(resultXml, wptserverOfResult, jobId)
+            persistJobResult(resultXml, jobId)
+            persistResultsForAllTeststeps(resultXml, jobId)
+            informDependents(jobId, resultXml)
```

Comment:
It would be nice if you can swap the parameter order to be streamlined with the other functions (`resultXml`, `jobId`)